bgpd/bgp_packet.c

```diff
@@ -1353,6 +1353,16 @@ static int bgp_open_receive(struct peer *peer, bgp_size_t size)
 		return BGP_Stop;
 	}
 
+	/* Send notification message when Hold Time received in the OPEN message
+	 * is smaller than configured minimum Hold Time. */
+	if (holdtime < peer->bgp->default_min_holdtime
+	    && peer->bgp->default_min_holdtime != 0) {
```

Comment:
Why do we need to check against 0 ?

Reply:
When minimum-holdtime value is 0, minimum-holdtime is disabled.